code/game/objects/items/robot/robot_items.dm

```diff
@@ -756,3 +756,73 @@
 /obj/item/borg/sight/hud/sec/Initialize(mapload)
 	. = ..()
 	hud = new /obj/item/clothing/glasses/hud/security(src)
+
+/**********************************************************************
+						Grippers
+***********************************************************************/
+
+/obj/item/weapon/gripper
+	name = "circuit gripper"
+	desc = "A simple grasping tool for inserting circuitboards into machinary."
```

Comment:
machinary

suggested: `"A simple grasping tool for inserting circuit boards into machinery."`